app/src/main/java/org/mozilla/fenix/components/InflationAwareFeature.kt

```diff
@@ -49,7 +50,11 @@ abstract class InflationAwareFeature(
      * when the view is inflated.
      */
     override fun start() {
-        // We don't do anything because we only want to start the feature when it's being used.
+        // We usually don't do anything because we only want to start the feature when it's being used.
+
+        // onStop() would basically leak FindInPageFeature which would remain displayed on screen
+        // but without registering any user interaction anymore.
+        (feature as? FindInPageFeature)?.start()
```

Comment:
Any reason we can't just do `feature?.start()`?

The `InflationAwareFeature` is abstract of any specific feature, so this would have issues with other features that will extend `InflationAwareFeature` as well.

Reply:
👍 I agree. Looking at this code `feature?.start()` seems to be the right thing to do here. Is this used by other features?

Reply:
> Is this used by other features?

Not yet, I think @colintheshots is looking into more UI features that can use more lazy view inflations.

Reply:
For some (like find in page) we probably should just make the feature work with a `ViewStub` instead of a `View`. That may be less complicated (code-wise) than wrapping the feature.